README.md

```diff
@@ -1,70 +1,66 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Bookstore
+"Bookstore" is a website for Books. It is a Single Page App (SPA) that allows users to:
 
-## Available Scripts
+* Add a book.
+* Track progress of reading.
 
-In the project directory, you can run:
 
-### `npm start`
+## Built With
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- Major languages<br/>
+    HTML, CSS, JavaScript
+- Frameworks<br/>
+    React
+- Technologies used
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Live Demo
```

Comment:
### Optional
- It's important to add a valid live demo link for your app. However, if it's not ready yet! You can mark it as `Coming soon` or something similar. 🙏